proto/lskvserver.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+import "etcd.proto";
+
+package lskvserverpb;
+
+message ReceiptClaims {
+  oneof request {
+    etcdserverpb.PutRequest request_put = 1;
+    etcdserverpb.DeleteRangeRequest request_delete_range = 2;
+    // TxnRequest request_txn = 3;
```

Comment:
Why is this not included in this PR?

Reply:
Fixed now we have the `set_allocated_*` working